infra-templates/core-services/v2.0.0-alpha1/compose.yml

```diff
@@ -12,7 +12,7 @@ services:
     privileged: true
     network_mode: host
     pid: host
-    command: plugin-manager --metadata-address 169.254.169.250 --disable-routesync --disable-cni-setup --metadata-listen-port 9346{{ if and (eq .Cluster.Orchestration "kubernetes") (ne .Cluster.Embedded "true") }} --disable-macsync --disable-arpsync --disable-conntracksync --disable-vethsync{{ end }}
+    command: plugin-manager --metadata-address 169.254.169.250 --disable-routesync --disable-cni-setup --disable-dns-setup --metadata-listen-port 9346{{ if and (eq .Cluster.Orchestration "kubernetes") (ne .Cluster.Embedded "true") }} --disable-macsync --disable-arpsync --disable-conntracksync --disable-vethsync{{ end }}
```

Comment:
`--disable-dns-setup` should only be for "kubernetes" orchestration type. can you add templating around it to check that `{{if (eq .Cluster.Orchestration "kubernetes")}} --disable-dns-setup {{ end}}`